service-base/src/main/java/org/eclipse/hono/service/AbstractServiceBase.java

```diff
@@ -315,6 +317,24 @@ protected final void addTlsTrustOptions(final NetServerOptions serverOptions) {
         }
     }
 
+    /**
+     * Configures the TLS engine for the given set of server options.
+     *
+     * @param serverOptions The options to add configuration to.
+     */
+    protected final void addTlsEngineOptions(final NetServerOptions serverOptions) {
```

Comment:
I think we should do this in the existing *addTlsKeyCertOptions* method so that we do not need to add another method call to all the adpters/components.

Reply:
Agreed, should I also consider changing the name or just put the lines to the existing method?

Reply:
the latter, please

Reply:
Adapted change